### DISCUSSION TITLE
Website not allowing me to login.

### DISCUSSION THREAD

**Discussion by di0xazin-e** [elrumo/macOS_Big_Sur_icons_replacements#1428] [Help with the website]
Hello! I am pretty sure it is just me but I went to go log into my account and for some reason, it just keeps loading and loading and does not allow me to log in. Hopefully there is a fix for me to do as I already did a-lot to try and get it work. Any help is very appreciated. :)

**Comment by atomtoto** [comment]
Same, and a lot of people have the same problem. (I ask on the macOSicon Discord)

**Reply by di0xazin-e** [reply]
Ah, I see. Well damn, is there any current information on progress to fix the bug? I assume since it just happened that they are still working on it.
